src/components/PreviewFrame.jsx

```diff
@@ -93,6 +89,13 @@ class PreviewFrame extends React.Component {
     });
   }
 
+  _updateSource(source) {
```

Comment:
Not a huge deal but I think it’s a bit strange to define this as an instance method—would make more sense to me to just have the logic inlined in the `onReady` below.